README.md

```diff
@@ -118,24 +158,28 @@ So, you can consider 1 as a good default value; it's strongly discouraged to set
 
 ### How the cache works
 
-`OwmCacheClient`'s internal cache follows this structure:
+`OwmCacheClient` and `DarkSkyCacheClient`'s internal cache stores responses in a map, which key is
+computed roughly as:
 
-|  Key                                                  |  Value            |
-| ----------------------------------------------------- | ----------------- |
-|  `round(latitude), round(longitude), beginningOfDay`  |  `List[Weather]`  |
+`round(latitude), round(longitude), beginningOfDay` 
 
 After the first request, it will try to fetch information for this exact place (round(latitude), round(longitude)) for the given day.
-This cache value usually consist of several (8-24) `Weather` objects, each with its own timestamps. `getCachedOrRequest` will return closest weather for specified timestamp and leave other in cache.
+The OWM cache value usually consist of several (8-24) `Weather` objects, each with its own timestamps.
 
+`cachingHistoryByCoords` in the OWM client will return closest weather for specified timestamp and leave other in cache.
```

Comment:
__the__ closest weather for __the__ specified timestamp and leave __the__ other__s__ in __the__ cache